scripts/publish_postgres

```diff
@@ -33,105 +38,152 @@ for index in ${!ARGS[*]}; do
   declare "${ARGS[$index]}"="${ARG_VALUES[$index]}"
 done
 
-
-TEMP_PATH=gs://${TEMP_BUCKET}/pipe-events/$( date -u "+%FT%T.%N" )
-
 echo "Running $0"
-echo "  START_DATE: $START_DATE"
+echo "  START DATE: $START_DATE"
+echo "  END DATE: $END_DATE"
 echo "  SOURCE: $SOURCE"
 echo "  TEMP_BUCKET: $TEMP_BUCKET"
-echo "  DEST_INSTANCE: $DEST_INSTANCE"
-echo "  DEST_CONNECTION_STRING: $DEST_CONNECTION_STRING"
-echo "  DEST_TABLE: $DEST_TABLE"
+echo "  DEST_PROJECT_ID: $DEST_PROJECT_ID"
+echo "  DEST_DB_REGION: $DEST_DB_REGION"
+echo "  DEST_DB_INSTANCE_NAME: $DEST_DB_INSTANCE_NAME"
+echo "  DEST_DB_USER: $DEST_DB_USER"
+echo "  DEST_DATABASE: $DEST_DATABASE"
+echo "  DEST_DB_PASSWORD: ********"
+echo "  DEST_DB_TABLE: $DEST_DB_TABLE"
 echo "  DEST_EVENT_TYPE: $DEST_EVENT_TYPE"
+
+
 ################################################################################
-# Export events to json files
+# Create required variables from params
 ################################################################################
-echo "Exporting events from $SOURCE"
-EXTRACT_PATH=$TEMP_PATH/bq/*.json
-bq extract \
-  --destination_format=NEWLINE_DELIMITED_JSON \
-  $SOURCE \
-  $EXTRACT_PATH
-if [ "$?" -ne 0 ]; then
-  echo "  Unable to extract ${SOURCE} to ${EXTRACT_PATH}"
-  exit 1
-fi
-echo "  Exported events from ${SOURCE} to ${EXTRACT_PATH}"
+start_date_formatted="${START_DATE//-/$''}"
+end_date_formatted="${END_DATE//-/$''}"
+temporal_table=${SOURCE}_${start_date_formatted}_${end_date_formatted}
+query_source="${SOURCE//:/$'.'}"
```

Comment:
For temporal tables we usually use `uuidgen` to build a unique temp table name, which we then delete after exporting to gcs. See for example https://github.com/GlobalFishingWatch/pipe-carrier-portal/blob/master/scripts/publish_vessel_tracks#L52-L54

Reply:
Done!